study-builder/studies/atcp/study.conf

```diff
@@ -1239,6 +1239,180 @@
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_STATUS",
+        "status": "ConsentedNeedsAssent"
+      },
+      "preconditionExpr": """user.studies["atcp"].forms["ASSENT"].hasInstance()""",
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_STATUS",
+        "status": "Consented"
+      },
+      "preconditionExpr": """!user.studies["atcp"].forms["ASSENT"].hasInstance()""",
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 3
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONTACTING_PHYSICIAN",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_STATUS",
+        "status": "SubmittedPhysicianInfo"
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "MEDICAL_HISTORY",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_STATUS",
+        "status": "SubmittedMedicalHistory"
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENOME_STUDY",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_STATUS",
+        "status": "SubmittedGenomeStudyShippingInfo"
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "REVIEW_AND_SUBMISSION",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_STATUS",
+        "status": "SubmittedEnrollment"
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 3
+    }
+    #REGISTRATION_TYPE events for DSM
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "REGISTRATION",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_TYPE",
+        "status": "Self"
+      },
+      "preconditionExpr": """operator.studies["atcp"].forms["PREQUAL"].questions["PREQUAL_SELF_DESCRIBE"].answers.hasOption("DIAGNOSED")"""
```

Comment:
Are we sure it's the `operator` and not the participant `user`?

Reply:
looks like A-T uses operator for all prequal related PEX checks